codelists/models.py

```diff
@@ -562,12 +563,30 @@ def table_with_fixed_headers(self):
         # Identify the index for the two columns we want
         code_header_index = header_row.index(code_header)
         term_header_index = header_row.index(term_header) if term_header else None
+
+        table_rows = self.table[1:]
+        if include_mapped_vmps and self.coding_system_id == "dmd":
+            term_header_index = term_header_index or len(self.table[0])
+            # ignore include_mapped_vmps if coding system is anything other than dmd
+            codes = [row[code_header_index] for row in table_rows]
+            # add in mapped VMP codes
+            previous_vmps_for_this_codelist = vmp_ids_to_previous(codes)
+            for current_vmp, previous_vmp in previous_vmps_for_this_codelist:
+                assert current_vmp in codes
+                if previous_vmp not in codes:
+                    new_row = ["" for i in range(len(table_rows[0]) + 1)]
+                    new_row[code_header_index] = previous_vmp
+                    new_row[
+                        term_header_index
+                    ] = f"Mapped previous VMP for {current_vmp}"
```

Comment:
Not sure this is the best wording.  "Superceded by {current_vmp}" maybe?

Reply:
The nomenclature in the source file and its metadata is "previous" ( e.g. `VPIDPREV` )

Reply:
Users don't necessarily know or need to know that though, do they?

Reply:
No, but (maybe just because I've been immersed in it!) I think that terminology is sufficiently clear and there is merit to matching what other sources call it

Reply:
Actually reviewing the code please disregard my comment I like what you've done there!